packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -1,79 +1,49 @@
 import { MutationTypes, CozyLink, getDoctypeFromOperation } from 'cozy-client'
 import PouchDB from 'pouchdb'
 import PouchDBFind from 'pouchdb-find'
-import fromPairs from 'lodash/fromPairs'
 import omit from 'lodash/omit'
+import defaults from 'lodash/defaults'
 
 import { withoutDesignDocuments } from './helpers'
 import { getIndexNameFromFields, getIndexFields } from './mango'
-
+import * as jsonapi from './jsonapi'
+import PouchManager from './PouchManager'
 PouchDB.plugin(PouchDBFind)
 
-const pipe = fn => res => {
-  try {
-    fn(res)
-  } catch (e) {
-    console.warn('Error during pipe', e)
-  }
-  return res
-}
+const sanitized = doc => omit(doc, '_type')
```

Comment:
ES6 new syntax let us do this without the need of a third-party library:

```js
const sanitized = ({_type, ...doc}) => doc
```

Is there a particular reason for using this third-party library?

Reply:
I find the intention clearer with `omit` here and `omit` has saner defaults IMHO.